queries.sql

```diff
@@ -56,3 +56,35 @@ SELECT MIN(weight_kg), MAX(weight_kg) FROM animals WHERE species = 'pokemon';
 
```

Comment:
- [ ] Also we are missing 2 of the screenshots of the query results. Kindly find the missing ones under the screenshot below and ads them accordingly
![Screenshot from 2023-08-17 09-02-20](https://github.com/rbd3/vet-clinic/assets/46388113/1928764f-c4ec-4da7-b3b8-a0f186018a4c)
![Screenshot from 2023-08-17 08-55-15](https://github.com/rbd3/vet-clinic/assets/46388113/627d982c-c80c-4f30-8c15-9a3fc94bcb9e)

Reply:
### Add screenshoot

![melody-pond](https://github.com/rbd3/vet-clinic/assets/119520276/aab68fb1-ce26-46b9-9610-5355cb46dfa1)
![most-animals](https://github.com/rbd3/vet-clinic/assets/119520276/e07f1911-9bcd-4fbc-8dc8-46b03d8f0024)